content/en/docs/collector/deployment/gateway.md

```diff
@@ -0,0 +1,170 @@
+---
+title: Gateway
+description:
+  Why and how to send signals to a single OTLP end-point and from there to
+  backends
+weight: 3
+---
+
+The gateway collector deployment pattern consists of applications (or other
+collectors) sending telemetry signals to a single OTLP endpoint provided by one
+or more collector instances running as a standalone service (for example, a
+deployment in Kubernetes), typically per cluster, per data center or per region.
+
+In the general case you can use an out-of-the-box load balancer to distribute
+the load amongst the collectors:
+
+![Gateway deployment concept](../../img/gateway-sdk.svg)
+
+For use cases where the processing of the telemetry data processing has to
+happen in a specific collector, you would use a two-tiered setup with a
+collector that has a pipeline configured with the [Trace ID/Service-name aware
+load-balancing exporter][lb-exporter] in the first tier and the collectors
+handling the scale out in the second tier. For example, you will need to use the
+load-balancing exporter when using the [Tail Sampling
+processor][tailsample-processor] so that all spans for a given trace reach the
+same collector instance where the tail sampling policy is applied.
+
+Let's have a look at such a case where we are using the load-balancing exporter:
+
+![Gateway deployment with load-balancing exporter](../../img/gateway-lb-sdk.svg)
+
+1. In the app, the SDK is configured to send OTLP data to a central location.
+1. A collector configured using the load-balancing exporter that distributes
+   signals to a group of collectors.
+1. The collectors are configured to send telemetry data to one or more backends.
+
+> **Note** Currently, only trace pipelines are supported by the load-balancing
+> exporter.
+
+## Example
+
+For a concrete example of the centralized collector deployment pattern we first
+need to have a closer look at the load-balancing exporter. It has two main
+configuration fields:
+
+- The `resolver`, which determines where to find the downstream collectors (or:
+  backends). If you use the `static` sub-key here, you will have to manually
+  enumerate the collector URLs. The other supported resolver is the DNS resolver
+  which will periodically check for updates and resolve IP addresses. For this
+  resolver type, the `hostname` sub-key specifies the hostname to query in order
+  to obtain the list of IP addresses.
+- With the `routing_key` field you tell the load-balancing exporter to route
+  spans to specific downstream collectors. If you set this field to `traceID`
+  (default) then the Load-balancing exporter exports spans based on their
+  `traceID`. Otherwise, if you use `service` as the value for `routing_key`, it
+  exports spans based on their service name which is useful when using
+  connectors like the [Span Metrics connector][spanmetrics-connector]], so all
```

Comment:
```suggestion
  connectors like the [Span Metrics connector][spanmetrics-connector], so all
```

Also, I generally favor sticking to the link text as a link name, so something like this:

```markdown

 ... connectors like the [Span Metrics connector][], so all ...

[span metrics connector]:
  https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/connector/spanmetricsconnector
```

(Especially when there's a single use of the link definition.)